### DISCUSSION TITLE
Upload file: Increase defaullt

### DISCUSSION THREAD

**Discussion by amin-senouci** [requarks/wiki#3211] [Help / Questions] [answered]
Hello team,

Someone can tell me how to increase the default file size supported by the upload file feature please? The default value (5M) is too small for my needs.
I am using:
OS: Ubuntu 18.04 and Docker
Wiki.js version: 2.5.170
Database engine: PostgreSQL 11.8

The recommendation in the documentation is deprecated for my wiki.js version: https://docs.requarks.io/install/config 
Thank you in advance for your help :)

Best regards
SEN

**Comment by NGPixel** [comment, marked as answer]
As stated in the docs, this is done via the admin area. Under Security.

**Reply by chloroplastgames** [reply]
Even if you configure limits other than the default ones, the content uploader continues notifing the default limits (Max 10 files, 5 MB each).

**Reply by NGPixel** [reply]
This is a UI bug only, the values are correctly set.
